Makefile

```diff
@@ -74,51 +75,61 @@ __check_defined = \
 ## End-to-end (E2E) Testing ##
 ##############################
 
+# Include project name in image name so that image name
+# does not conflict with other images during local development
+E2E_IMAGE_NAME := $(PROJECT_ROOT)-$(APP_NAME)-e2e
+
 e2e-build: ## Build the e2e Docker image, if not already built, using ./e2e/Dockerfile
-	docker build -t playwright-e2e -f ./e2e/Dockerfile .
+	docker build -t $(E2E_IMAGE_NAME) -f ./e2e/Dockerfile .
 
-e2e-clean-report: ## Remove the local ./e2e/playwright-report and ./e2e/test-results folder and their contents
+e2e-clean-report: ## Remove the local e2e report folders and content
 	rm -rf ./e2e/playwright-report
 	rm -rf ./e2e/blob-report
 	rm -rf ./e2e/test-results
+	rm -rf ./e2e/example-screenshot.png
```

Comment:
generalize this to work for all screenshots not just the example test that's included in the tempalte

Reply:
Just fyi the screenshots are typically only generated in the `./e2e/playwright-report` folder - but since we have a test that specifically takes a screenshot - it pops up at `./e2e/example-screenshot.png` . However we can generalize to just remove all `.png` 

```
  test('should take a screenshot of the webpage', async ({ page }) => {
    await page.goto('/');
    await page.screenshot({ path: 'example-screenshot.png', fullPage: true });
  });
 
  ```

Reply:
Actually the report and screenshot seem to be generated in different places which doesn't quite seem right - opened a separate ticket to look into that https://github.com/navapbc/template-infra/issues/800